settings.gradle

```diff
@@ -1,3 +1,6 @@
+plugins {
+    id 'org.gradle.toolchains.foojay-resolver-convention' version '0.8.0'
```

Comment:
Unless there is a compelling reason why we need this, please can you remove this plugin.  We already have `.sdkmanrc` to help pull down the appropriate jdk for the branch (`sdk env`).